src/TextField.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cx from 'classnames';
+import Label from './Label';
+
+/**
+ * @module TextField
+ */
+class TextField extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value,
+			hasError: props.error,
+			error: props.error
+		};
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	handleChange(e) {
+		this.setState({ value: e.target.value });
+	}
+
+	render() {
+		const {
+			inputId,
+			name,
+			label,
+			labelClassName,
+			placeholder,
+			formAttrs,
+			className,
+			children,
+			...other
```

Comment:
Since you're passing `...other` to the `<input>`, I think it might be simpler to leave out any mention of `placeholder` and `name` - they will be populated correctly by `other`. Similarly, you can remove `inputId` and just let the parent component specify `id`, which will be applied to the `<input>` by `other`. The fewer props that have to be renamed before they get to the HTML output, the better.